example/python/tx-example.py

```diff
@@ -156,6 +156,28 @@ def tx4():
     send_tx(tx, key_pair)
     print_status_streaming(tx)
 
+def tx5():
```

Comment:
Better to add documentation string with a description of use case. The best choice here use `given, when, then` notation. Same with the next example.
Also, the name looks doubtful. Maybe change it with more suitable and nice for a user?

Reply:
It is not a test. We don't have checks on expected behaviour